### ISSUE TITLE
Mod doesn't seem to work at all...

### ISSUE THREAD

**Issue by Bunglar** [WondaMegapon/Refightilization#38]
So, getting some weird behavior from it. Playing with someone else and when we'd die, we wouldn't respawn and we'd just spectate as normal... BUT when we BOTH died, then I respawn as a monster during the results screen...? It is worth noting I had a bunch of other mods and at least one may have conflicted with it, but I started a new profile in R2 mod manager that was ONLY this mod, and the issue persisted.

**Comment by WondaMegapon** [comment]
Oh? Why was this closed?

**Comment by Bunglar** [comment]
Sorry for INCREDIBLY late reply, honestly forgot about this. So yeah turns out it was compatibility issues. I found out for some reason this mod doesn't work with certain characters and pretty much any skin mod for some reason. So enforcer gang and paladin work fine, but I tried Tesla Trooper and he breaks refightilization. In addition I tried some skins that all seemed to break it too. For some reason there were some pokemon skins from TailLover that all seemed to work with refightilization just fine but I wanted the actual good skins like pepsiman (lol) then refight would stop working. So yeah I closed it because I figured out why it wasn't working and I didn't respond after so long because I closed it and wasn't expecting a response, sorry.
